ignite/distributed/utils.py

```diff
@@ -737,3 +738,17 @@ def download_dataset():
 
     if current_rank == rank:
         barrier()
+
+
+def _rank_not_in_group(group: Optional[Union[Any, List[int]]]) -> bool:
+    """Check if the current process's rank is not in a given group."""
+    if group is None:
+        return False
+    if isinstance(group, list) and all(isinstance(item, int) for item in group):
+        group = new_group(group)
+    if has_hvd_support:
+        from hvd.common.process_sets import ProcessSet
+
+        if isinstance(group, ProcessSet):
+            return group.included()
+    return dist._rank_not_in_group(group)
```

Comment:
Ideally, we should implement `_rank_not_in_group` in all backends and call `_model._rank_not_in_group` method from here.

Reply:
Yes, you're right